pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -197,14 +197,25 @@ class GammaGammaModel(BaseGammaGammaModel):
             from pymc_marketing.clv import GammaGammaModel
 
             model = GammaGammaModel(
-                customer_id=[0, 1, 2, 3, ...],
-                mean_transactionn_value=[23.5, 19.3, 11.2, 100.5, ...],
-                frequency=[6, 8, 2, 1, ...],
-                p_prior=pm.HalfNormal.dist(10),
-                q_prior=pm.HalfNormal.dist(10),
-                v_prior=pm.HalfNormal.dist(10),
+                data=pd.DataFrame({
+                    "customer_id": [0, 1, 2, 3, ...],
+                    "mean_transactionn_value" :[23.5, 19.3, 11.2, 100.5, ...],
+                    "frequency": [6, 8, 2, 1, ...],
+                }),
+                model_config={
+                    "p_prior": {dist: 'HalfNorm', kwargs: {}},
+                    "q_prior": {dist: 'HalfStudentT', kwargs: {}},
+                    "v_prior": {dist: 'HalfCauchy', kwargs: {}},
```

Comment:
This is invalid, as the HalfStudentT for sure requires at least `df`. I suggest to just keep the original example with `HalfNormal(sigma=10)` for everything

Reply:
I've replaced it empty kwargs dict with {"nu": 4, "sigma": 10} as in other examples